build-support/functions/20-build.sh

```diff
@@ -275,7 +274,28 @@ function build_consul {
       extra_dir="${extra_dir_name}/"
    fi
 
-   local container_id=$(docker create -it -e CGO_ENABLED=0 ${image_name} gox -os="${XC_OS}" -arch="${XC_ARCH}" -osarch="!darwin/arm !freebsd/arm !darwin/arm64" -ldflags "${GOLDFLAGS}" -output "pkg/bin/${extra_dir}{{.OS}}_{{.Arch}}/consul" -tags="${GOTAGS}")
+   # figure out if the compiler supports modules
```

Comment:
Would you prefer to just hardfail if someone has modules disabled now? Or are will kicking that can for later?

Reply:
I was kicking that can down the road. We could do it unconditionally though.